server.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+const net = require('net');
+const EE = require('events');
+const Client = require('./model/client.js');
+const PORT = process.env.PORT || 3000;
+const server = net.createServer();
+const ee = new EE();
+
+const pool = [];
+
+ee.on('@dm', (client, string) => {
+  let nickname = string.split(' ').shift().trim();
+  let message = string.split(' ').splice(1).trim();
+
+  pool.forEach((client) => {
+    if(client.nickname === nickname)
+      client.socket.write(`${client.nickname}: ${message}`);
+  });
+});
+
+ee.on('@all', (client, string) => {
+  pool.forEach((c) => {
+    c.socket.write(`${client.nickname}: ${message}`);
```

Comment:
message is changed from message to string. I did this because we didnt set the string to a var like message like we did the block up.